src/Manifolds.jl

```diff
@@ -110,6 +111,7 @@ using ManifoldsBase: QRRetraction, PolarRetraction, ProjectionRetraction
 using ManifoldsBase: AbstractInverseRetractionMethod, LogarithmicInverseRetraction
 using ManifoldsBase: QRInverseRetraction, PolarInverseRetraction, ProjectionInverseRetraction
 using ManifoldsBase: AbstractVectorTransportMethod, ParallelTransport, ProjectionTransport
+using ManifoldsBase: ℝ, ℂ, ℍ
```

Comment:
Mabe you can add `real_dimension` as an import here, too?

Reply:
ManifoldsBase.jl exports `real_dimension` anyway and as far as I can see it's not extended with new methods in Manifolds.jl.

Reply:
Sorry then I was imprecise, I meant to add it to this `using` (not import) because I think we use it in manifolds.

Reply:
We already have `using ManifoldsBase` so the symbol `real_dimension` is visible anyway.